tests/test-cases-lib.sh

```diff
@@ -7,10 +7,42 @@ function TEST_all() {
   TEST_markov-chain-generator $demo_root
   TEST_road-traffic-predictor $demo_root
   TEST_tensorflow $demo_root
+  TEST_realtime-trade-monitor $demo_root
+  TEST_flight-telemetry $demo_root
 }
 
 function TEST_flight-telemetry() {
-  echo "********** TEST NOT IMPLEMENTED YET: flight-telemetry **********"
+  echo "********** START TEST: flight-telemetry **********"
+  local demo_root=$1
+  local log_file=flight-telemetry.txt
+  local pause=20
+
+  echo "********** Start flight-telemetry project with 200 sec timeout **********"
+  cd $demo_root/flight-telemetry
+  make up
+  sleep $pause
+  timeout 200 mvn exec:java -U -B | tee ${log_file}&
+  local FLIGHT_TM_PID=$!
+  sleep $pause
+
+  echo "********** Launch data-observer to make sure landingMap is being updated by job **********"
+  cd $demo_root/tests/tools/qe-data-observer/target
+  java -jar qe-data-observer-5.0-SNAPSHOT-jar-with-dependencies.jar FlightTelemetry landingMap
+  local TEST_RESULT=$?
+
+  #  cleaning up the environment - killing all launched processes
+  kill -KILL $FLIGHT_TM_PID
+  make down
+
+  if [ "$TEST_RESULT" == "1" ]
```

Comment:
I think generally we should rather check the output was correct (i.e. whether it was `0`) than it was incorrect - what if the command finished with some other error code than `1`?

Reply:
Really... Nice improvement. Changed.